containers/elm-0.19.0/test-project/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014-2016, Evan Czaplicki
```

Comment:
We can't accept contributions using alternate licenses because of the complexity of distributing them unfortunately.  Is the license required here?

Reply:
The test-project folder is a straight clone of the [elm-architecture-tutorial repository ](https://github.com/evancz/elm-architecture-tutorial), which requires this license to be distributed. If we can't include it, I could remove that test-project entirely and make one from scratch that's a simple Hello World?

It wouldn't be as complete, but probably good enough to make sure the dev tools are working properly.

Reply:
@xWiiLLz Yeah I think that would be best given the license is required in the other example. Thanks!!